tests/unit/commands/test_cmdutils.py

```diff
@@ -397,6 +397,16 @@ def fun(arg=0):
 
         assert str(excinfo.value) == "Argument marked as both count/win_id!"
 
+    def test_zero_count_and_win_id_same_arg(self):
+        with pytest.raises(TypeError) as excinfo:
+            @cmdutils.argument('arg', count=True,
+             zero_count=True, win_id=True)
+            def fun(arg=0):
+                """Blah."""
+                pass
+
+        assert str(excinfo.value) == "Argument marked as both count/win_id!"
+
```

Comment:
See my comment above - the test should check `zero_count=True` without `count=True` - this test as-is doesn't really test anything new.